CHANGELOG.md

```diff
@@ -1,21 +1,26 @@
 # CHANGELOG
 
+## UNRELEASED
+
+* Added Swift 3 support (Xcode 8 beta 2) while retaining Swift 2 support.
+  [@phatblat](https://github.com/phatblat), [#20](https://github.com/AliSoftware/Reusable/pull/20)
+
 ## 2.3.0
 
-* Added ` NibOwnerLoadable` protocol for `UIView` set as XIB's File's Owner.  
```

Comment:
Please do NOT remove the two spaces after the full stop before the newline, so that it insert a newline in the same paragraph in the rendered markdown.